### PR TITLE
Feature/add requests resource

### PR DESCRIPTION
**Pull request by klawani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

add resource request to test-service

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: nodejs/templates/tests/test-service.yaml [hunk 1]
```diff
@@ -19,4 +19,11 @@ spec:
         echo 'Testing liveness';
         livenessStatusCode=$(wget -S http://$SERVICE_NAME/health/liveness 2>&1 | grep HTTP/ | awk 'END{print $2}');
         test "$livenessStatusCode" = "200"; echo $?;
+    resources:
```
Comment (adusumillipraveen) [inline nodejs/templates/tests/test-service.yaml:22]: Not sure if its worth getting them from values.

File: nodejs/templates/tests/test-service.yaml [hunk 2]
```diff
@@ -19,4 +19,11 @@ spec:
...
+      limits:
+        cpu: 2500m
```
Comment (adusumillipraveen) [inline nodejs/templates/tests/test-service.yaml:24]:
```suggestion
        cpu: 1
```

File: nodejs/templates/tests/test-service.yaml (hunk 1 above)
Comment (klawani) [reply nodejs/templates/tests/test-service.yaml:22]: done

**Comment by klawani** [comment]
close it and want to open a new one

**Pull request closed without merging**